runtime/dotnet/azurewebapp/Startup.cs

```diff
@@ -84,9 +85,14 @@ public IStorage ConfigureStorage(BotSettings settings)
             return storage;
         }
 
+        public bool ShouldUseAuthorizationMiddleWare(BotSettings settings)
+        {
+            return settings?.Feature?.UseAuthorizationMiddleware == true;
```

Comment:
Naming: Features should be higher level feature ideally. A more high level, user facing feature I'm thinking here would be _is Is this a Skill?_ or _Is this a Skill Host?_. In that context, we could have 2 settings: IsSkill and IsSkillHost, and when IsSkill is true, then one of the consequences is that we enable this authorization claims validator.

I encourage you to provide feedback on this suggestion, would love to hear your thoughts on it.